src/main/java/net/dv8tion/jda/api/utils/WidgetUtil.java

```diff
@@ -534,7 +534,7 @@ public boolean equals(Object obj) {
         @Override
         public String toString()
         {
-            final EntityString entityString = new EntityString(this);
+            final EntityString entityString = new EntityString("Widget");
```

Comment:
I wonder if we should move the Widgets entities out of this Util file and into the `.entities` package under `.entities.widget`

@MinnDevelopment, thoughts?

Reply:
(this could definitely be done outside of this PR, just wanted to surface)

Reply:
That would probably make sense, does the widget util still work? It hasn't really been touched in years

Reply:
i haven't used it in years, but I'd assume it does.